draft-ietf-intarea-provisioning-domains.md

```diff
@@ -562,11 +562,12 @@ while using exclusively the set of configuration information attached
 with the PvD, as defined in {{host}}. In some cases, it
 may therefore be necessary to wait for an address to be available for
 use (e.g., once the Duplicate Address Detection or DHCPv6 processes
-are complete) before initiating the HTTP over TLS query. If the host
-has a temporary address per {{!RFC4941}} in this PvD, then
-hosts SHOULD use a temporary address to fetch the PvD Additional
-Information and SHOULD deprecate the used temporary address and
-generate a new temporary address afterward.
+are complete) before initiating the HTTP over TLS query. In order to
+address privacy concerns around linkability of the PvD HTTP connection
+with future user-initiated connections, if the host has a temporary address
+per {{!RFC4941}} in this PvD, then it SHOULD use a temporary address
+to fetch the PvD Additional Information and SHOULD deprecate the used
```

Comment:
I think the second SHOULD could be a MAY.

Reply:
> I think the second SHOULD could be a MAY.

Sorry to disagree, it should be a SHOULD

Reply:
Don't be sorry ;-)
Maybe I am missing something.
AFAIU, all the operator knows when it receives the PvD HTTPs GET is that somebody is there (there is no further identification when doing that query).
And then later, if we see the same address doing another query somewhere else, we can surely tie it to the original PvD GET, but how is that leaking personal information ?

Reply:
;-)
For sure, multiple devices on the same PIO prefix will always share a /64 but they all appears as the same /64. The PvD GET will probably send some HTTP header (user-agent, charset, language, ...) to the network operator allowing the operator to 'kind of' guess the client for further IP traffic from that address and correlate them. Hence, the SHOULD (aim to do it but exceptions are allowed) to prevent this correlation.
Yes, sometimes I am paranoid ;-)

Reply:
I think the deprecating of the address is a bit harsh though for many implementations, and would be a bit of a burden to do. Regarding the HTTP get for the PvD, there's nothing saying that it will actually have the user-agent of further traffic.

I'm tempted to agree with @Oryon and make it a MAY. It describes the problem, and concerned parties can choose to do what they will with it.

Reply:
Switched to MAY for now